code/modules/mob/living/carbon/xenomorph/hive_datum.dm

```diff
@@ -431,10 +440,6 @@ to_chat will check for valid clients itself already so no need to double check f
 // ***************************************
 // *********** Normal Xenos
 // ***************************************
-/datum/hive_status/normal // subtype for easier typechecking and overrides
```

Comment:
Just typecast the hive like I did in #2490

Reply:
That's not a good idea. This makes hives more generic and makes us able to add this functionality to all other hives in the future.

Reply:
That will require a larger rework we currently have no plans for.

Reply:
But changing it right now doesn't break anything, does it?

Reply:
Eh, I suppose not. Though it sparks the discussion of the future of hives.